scripts/rff/preprocess_csv_test.py

```diff
@@ -0,0 +1,27 @@
+"""Tests for preprocess_csv.py"""
+
+import os
+import sys
+import tempfile
+import unittest
+
+# Allows the following module imports to work when running as a script
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from rff import preprocess_csv
+
+
+class ProcessTest(unittest.TestCase):
+
+    def test_preprocess(self):
+        src_fldr = "test_data/input/prism/daily/county"
+        expected_csv = "test_data/expected/WeatherVariability_Counties.csv"
```

Comment:
This test fails when run as part of presubmit (as below).

So I think we want something like this:

```
_TESTDIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'test_data')
src_fldr = os.path.join(_TESTDIR, 'input/prism/daily/county')
expected_csv = os.path.join(_TESTDIR, 'expected/WeatherVariability_Counties.csv')
```

Failure:

```
======================================================================
ERROR: test_preprocess (rff.preprocess_csv_test.ProcessTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/workspace/scripts/rff/preprocess_csv_test.py", line 21, in test_preprocess
    with open(expected_csv, 'rb') as wantf:
FileNotFoundError: [Errno 2] No such file or directory: 'test_data/expected/WeatherVariability_Counties.csv'

----------------------------------------------------------------------
```